website/docs/best-practices/how-we-build-our-metrics/semantic-layer-3-build-semantic-models.md

```diff
@@ -1,40 +1,37 @@
 pagination_next: "best-practices/how-we-build-our-metrics/semantic-layer-4-build-metrics"
 ---
 
 ## How to build a semantic model
 
-A semantic model is the MetricFlow equivalent to a logical layer model (what historically has just been called a 'model' in dbt land). Just as configurations for models are defined on the `models:` YAML key, configurations for semantic models are housed under `semantic models:`. A key difference is that while a logical model consists of configuration and SQL or Python code, a **semantic model is defined purely via YAML**. Rather than encoding a specific dataset, a **semantic model describes relationships** that let your end users select and refine their own datasets reliably.
+A semantic model is the Semantic Layer equivalent to a logical layer model (what historically has just been called a 'model' in dbt land). Just as configurations for models are defined on the `models:` YAML key, configurations for semantic models are housed under `semantic models:`. A key difference is that while a logical model consists of configuration and SQL or Python code, a **semantic model is defined purely via YAML**. Rather than encoding a specific dataset, a **semantic model describes relationships and expressions** that let your end users select and refine their own datasets dynamically and reliably.
 
 - ⚙️ Semantic models are **comprised of three components**:
   - 🫂 **entities**: these describe the **relationships** between various semantic models (think ids)
-  - 🪣 **dimensions**: these are the columns you want to **slice, dice, group, and filter by** (think timestamps, categories, booleans).
+  - 🔪 **dimensions**: these are the columns you want to **slice, dice, group, and filter by** (think timestamps, categories, booleans).
   - 📏 **measures**: these are the **quantitative values you want to aggregate**
-- 📚 We define **columns as being an entity, dimension, or measure**.
-
-:::tip
-**File per model**. Given the interdependence of logical and semantic models, and semantic models and metrics, we've updated our best practice recommendation to a one YAML file per model approach if you're using the Semantic Layer. This houses everything related to a model in one place and preserves unique file names for quickly getting to the code you want.
-:::
+- 🪣 We define **columns as being an entity, dimension, or measure**. Columns will typically fit into one of these 3 buckets, or if they're a complex aggregation expression, they might constitute a metric.
 
 ## Defining orders
 
-- 🥪 The semantic model we're going to define is _orders_.
-- 📗 We define it as a **YAML dictionary in the semantic models list**.
+Let's zoom in on how we might define an _orders_ semantic model.
+
+- 📗 We define it as a **YAML dictionary in the `semantic_models` list**.
 - 📑 It will have a **name, entities list, dimensions list, and measures list**.
 - ⏬ We recommend defining them **in this order consistently** as a style best practice.
 
```

Comment:
this is just a non blocker suggestion but I've started to wrap code around a `file name` component (?) bc i realized it helps users know where to save the file (some users opened up prs adding this specifically).

i also wonder if it'd be helpful to follow the best practices naming convention? https://docs.getdbt.com/best-practices/how-we-structure/5-semantic-layer-marts#naming-convention

```suggestion
<File name='models/marts/sem_orders.yml'>
```YAML
```

Reply:
oh nice re File component, will do!

re the How we structure stuff, realizing we need to update that as well, I'll add some commit with that to this so they go out together!